plugins/outputs/elasticsearch/README.md

```diff
@@ -323,6 +323,11 @@ to use them.
   ## no pipeline is used for the metric.
   # use_pipeline = "{{es_pipeline}}"
   # default_pipeline = "my_pipeline"
+  #
+  # Custom HTTP headers
+  # To pass extra HTTP headers please define it in a given below section
+  # [outputs.elasticsearch.extra_headers]
+  #    "X-Custom-Header" = "custom-value"
```

Comment:
Can we name this just `headers` please? I don't see where this is "extra"...

Reply:
just updated